<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="TimerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вкл./откл. время</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableFiles">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>501</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Inode</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип файла</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя файла</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="changeDirButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сменить каталог</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий каталог:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currdLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>441</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="makeFIFO">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать канал FIFO </string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteUFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить файл с уникальным именем</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createUFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать файл с уникальным именем</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Система</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Выйти</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Отладка</string>
    </property>
    <addaction name="menuDebug_onoff"/>
    <addaction name="MenuDB"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Информация о системе....</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Информация о разработчике....</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Выйти из программы</string>
   </property>
  </action>
  <action name="menuDebug_onoff">
   <property name="text">
    <string>Вкл./откл. отладку</string>
   </property>
  </action>
  <action name="MenuDB">
   <property name="text">
    <string>База данных</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
